File: python/chapter5/checking_usernames.py
current_users = ["john", "jane", "adam", "jack", "jenny", "barry", "joe"]
new_users = ["jonas", "jane", "Jenny", "angela", "jack"]

for user in new_users:
    if user.lower() in current_users:
        print("Username already taken, choose another name.")
    else:
        print("Username is available!")

File: javascript/chapter5/checking_usernames.js
const currentUsers = ["john", "jane", "adam", "jack", "jenny", "barry", "joe"];
const newUsers = ["jonas", "jane", "Jenny", "angela", "jack"];

for (let user of newUsers) {
	if (currentUsers.includes(user.toLowerCase())) {
		console.log("Username already taken, choose another name.");
	} else {
		console.log("Username is available!");
	}
}
